server/helpers.js

```diff
@@ -0,0 +1,78 @@
+const axios = require('axios'); //[https://github.com/axios/axios]
+const config = require('../config.js'); //provides API token for insertion into Authorization header
+
+//Products GET /products Retrieves the list of products
+//param: page (integer) Selects the page of results to return. Default 1.
+//param: count (integer) Specifies how many results per page to return. Default 5
+let getProductsList = () => {
+  let options = {
+    method: 'get',
+    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products`,
+    //url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products?page=1&count=5`,
```

Comment:
We should probably clean up the pseudocode here a bit, as well as on lines 

- [ ] 13
- [ ] 20-21
- [ ] 28-29
- [ ] 39
- [ ] 46-47
- [ ] 54-55

I can easily take care of it when next I edit the server code, though.

(I would have just made a general "clean up pseudocode" comment, but I wanted to practice comment formatting here!)